README.rst

```diff
@@ -91,6 +91,28 @@ that you have the ``universe`` repository enabled. The ``universe`` repository
 is enabled by default in a standard Ubuntu installation but may not be on some
 minimal, or not standard, installations.
 
+On Docker::
+
+Required dependencies installed on the target system:
+  - Docker
+  - docker-compose
+
+You can pull the image from our Docker Hub
```

Comment:
I think you need to put at the end of the line `::` so that it considers this a code block (as opposed to inline code)